app/src/main/java/com/onfido/android/app/sample/FinalActivity.java

```diff
@@ -40,7 +40,7 @@ public void onAlternateDocumentSelected() {
 
     public void showFinalScreen() {
         Fragment fragment = MessageFragment.createInstance(
```

Comment:
we should probably remove using the MessageFragment like this, since it's not part of our "Public api"

Reply:
what would be the replacement? Feel free to apply changes to the PR

Reply:
not as part of this PR (sorry for confusion). I will add an issue to tackle it later.

Reply:
it's under CX-1501

Reply:
there is one thing we can do. comment the code and inform the developer not to use this class since it's private

Reply:
let's tackle that in the next PR then